src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -170,6 +170,10 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
   @Query("Update SessionTemplate s set s.active = :isActive WHERE s.reference = :reference")
   fun updateActiveByReference(reference: String, isActive: Boolean): Int
 
+  @Modifying
+  @Query("Update SessionTemplate s set s.isTapSession = :isTapSession WHERE s.reference = :reference")
+  fun updateIsTapSessionByReference(reference: String, isTapSession: Boolean): Int
```

Comment:
should be in SESSION_LOCATION_GROUP table as an exclude, this needs to be data driven